.github/workflows/firesim-run-tests.yml

```diff
@@ -93,16 +94,20 @@ jobs:
     runs-on: ${{ github.run_id }}
     steps:
       - uses: actions/checkout@v2
+      - name: Install FireSim pem
+        run: ./.github/scripts/install-firesim-pem.py
       - name: Setup FireSim repo (.pem, build-setup.sh, AWS credentials, submodules) and CI daemons
-        uses: ./.github/actions/initialize-manager
+        run: ./.github/scripts/initialize-manager.py
```

Comment:
I think we should rename this script to something like `initialize-repo`. Saying `init-manager` to me implies that the entire machine is getting setup.

Reply:
I like that.